app/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
 	}
 
 	dependencies {
-		classpath 'com.android.tools.build:gradle:3.1.3'
+		classpath 'com.android.tools.build:gradle:3.2.1'
```

Comment:
there's a newer version 3.3.1

Reply:
There is, but it has compatibility issues with generated code, such as the `R` lookup file and some Kotlin-generated sources.